### PR TITLE
use multi-stage build in Dockerfile (fixes FINERACT-771)

### PR DESCRIPTION
**Pull request by vorburger**
also removing git & JDK from Dockerfile; see 2nd commit message

### CODE REVIEW

**Comment by vorburger** [comment]
@conradsp do you want to review, and perhaps (manually) test this, just to double check?

@awasum FYI.

File: Dockerfile [hunk 1]
```diff
@@ -15,24 +15,26 @@
...
 WORKDIR fineract
 RUN sed -i 's/localhost/fineractmysql/' ./fineract-provider/src/main/resources/sql/migrations/list_db/V1__mifos-platform-shared-tenants.sql
 RUN ./gradlew clean war
-RUN mv build/libs/fineract-provider.war /opt/bitnami/tomcat/webapps
+
+
+FROM bitnami/tomcat:7.0.94 as fineract
+
+USER root
+RUN apt-get update -qq && apt-get install -y git openjdk-8-jdk wget
```
Comment (awasum) [inline Dockerfile]: Why are you installing git? Are you using it somewhere?

Comment (awasum) [reply Dockerfile]: My bad...did not see the second commit..even as you highlighted in your message.

**Comment by conradsp** [comment]
Tested manually. All looks good. Merging.

**Pull request merged**